crates/sui-json-rpc/src/api.rs

```diff
@@ -476,6 +476,7 @@ pub trait TransactionExecutionApi {
     ///     makes sure this node is aware of this transaction when client fires subsequent queries.
     ///     However if the node fails to execute the transaction locally in a timely manner,
     ///     a bool type in the response is set to false to indicated the case.
+    // TODO: remove and prefer executeTransactionSerializedSig
```

Comment:
or remove this and rename `executeTransactionSerializedSig` to `executeTransaction`